worlds/manual_outerwildsdlc_nicopopxd/data/locations.json

```diff
@@ -106,86 +111,86 @@
 		"name": "4 - Visit the Planet's Core",
 		"region": "Space",
 		"category": [ "4 - Giant's Deep" ],
-		"requires": [ "Tornado Knowledge", "Jellyfish Knowledge" ]
```

Comment:
you removed jellyfish knowledge from going to Giant's deep's core, Going fast wasnt in my logic lol

Reply:
Fixed